### ISSUE TITLE
Exit URL no longer exists.  Causes study to break. 

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#842]
# TL;DR
The recent external studies feature included the removal of the `exit_url` field.  This has caused an issue with a researcher.  

# Narrative
This issue is in reference to [this slack thread](https://lookit-mit.slack.com/archives/CHG0M1LA1/p1635181469013000).  The obvious solution is to put the field back (probably as an empty field) and then as the researcher to replace their exit url link.  

I would argue that a ember frame player frame would be a better solution.  Create/use a frame that would end the study with a redirect to which ever link the researcher would prefer.  

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature

# Implementation Notes
I am leaving this up for discussion, but I am leaning towards the frame solution.  I am interested where this will go though.

**Comment by mekline** [comment]
@okaycj can we close this? I think we decided to keep the exit URL in the end?

**Comment by okaycj** [comment]
> 

Yes.
